keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -172,27 +172,48 @@ type PersistentVolumeClaimSpec struct {
   ...
 }
 
+// Add the PersistentVolumeClaimModifyVolume condition to existing PersistentVolumeClaimConditionType
+// Condition is used to document the last error controller sees
+const (
+  ...
+  // PersistentVolumeClaimVolumeAttriburesModifyError - a user trigger modify volume of pvc has error
+  PersistentVolumeClaimVolumeAttriburesModifyError PersistentVolumeClaimConditionType = "VolumeAttriburesModifyError"
```

Comment:
We also want to add the "VolumeAttributesModifying" condition based on this discussion with @gnufied https://github.com/kubernetes/enhancements/pull/4298#discussion_r1369227401

Reply:
Why do we need a modifying? We cannot just say "Err but we are retrying" in the messaging?

Reply:
How does an operator that is possibly monitoring the pvc knows volume is being modified or not? By regexp. for message on the condition? Seems fragile.